src/o365/spo/commands/list/list-webhook-list.ts

```diff
@@ -124,15 +142,23 @@ class SpoListWebhookListCommand extends SpoCommand {
     const options: CommandOption[] = [
       {
         option: '-u, --webUrl <webUrl>',
-        description: 'URL of the site where the list to retrieve webhooks for is located'
+        description: `URL of the site where the list to retrieve webhooks for is located`
+      },
+      {
+        option: '--listId [listId]',
+        description: `ID of the list to retrieve all webhooks for. Specify either id or title but not both`
```

Comment:
We should refer to the new options `listId` and `listTitle` instead of the deprecated `id` and `title`